src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -126,4 +130,36 @@ class AktivitetskravService(
             aktivitetskrav = updatedAktivitetskrav
         )
     }
+
+    suspend fun sendForhandsvarsel(
+        personIdent: PersonIdent,
+        veilederIdent: String,
+        aktivitetskravUuid: UUID,
+        forhandsvarselDTO: ForhandsvarselDTO,
+        callId: String,
+    ): AktivitetskravVarsel {
+        val aktivitetskrav =
```

Comment:
Synes kanskje det hadde vært penere og gjort de to sjekkene under i api-laget - siden vi også gjør sjekk på `document` der. Og da sende inn et `aktivitetskrav` som parameter hit i stedet for `personIdent` og `aktivitetskravUuid`.